Backend/src/main/java/lt/vu/ads/exceptions/ResourceNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package lt.vu.ads.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class ResourceNotFoundException extends Exception {
+    private static final long serialVersionUID = 1L;
+
+    public ResourceNotFoundException (String message){
+        super(message);
+    }
+}
```

Comment:
why it is needed to create new custom exception when we can just use throw new Exception(someStringMessage)?

Reply:
Also, Spring has built in HTTP error exceptions already.

Refer to [this](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/client/HttpClientErrorException.html)

Reply:
We want to throw unchecked exceptions, which when thrown inside a transaction, the transaction is rolled back, otherwise commited